### PR TITLE
Sanitize error output for prevent XSS security issues

### PR DESCRIPTION
**Pull request by msaf1980**

### CODE REVIEW

File: webapp/graphite/errors.py [hunk 1]
```diff
@@ -102,6 +111,6 @@
         except InputParameterError as e:
             msgStr = str(e)
             log.warning('%s', msgStr)
-            return HttpResponseBadRequest(msgStr)
+            return HttpResponseBadRequest(escape(msgStr))
```
[the remark this answers is not in the document]
Comment (msaf1980) [reply webapp/graphite/errors.py:114]: It's a problem ? Error message is sanitized and it's safer than now.

[the remark this answers is not in the document]
Comment (deniszh) [reply webapp/graphite/errors.py:114]: It's a warning, so, you can ignore it, I think.

**Comment by deniszh** [comment]
Hi @msaf1980 
Thanks for your patch! maybe you know, do we have issue for that xss? or it's new?

**Comment by msaf1980** [comment]
> Hi @msaf1980 Thanks for your patch! maybe you know, do we have issue for that xss? or it's new?

As I think, It's fix all XSS raised from handlers annotated with `@handleInputParameterError` (if exception is InputParameterError)

**Comment by deniszh** [comment]
Yes, looks like majority of issues. Nice fix! Will merge and check opened xss tickets against it.

**Comment by deniszh** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|1.1.x|[](https://github.com/graphite-project/graphite-web/pull/2799)|


### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**